cli/ingress-controller/flags.go

```diff
@@ -31,6 +33,20 @@ import (
 	"github.com/kong/kubernetes-ingress-controller/internal/ingress/controller"
 )
 
+type headers []string
+
+func (h *headers) String() string {
+	return "my string representation"
+}
+
+func (h *headers) Set(value string) error {
+	if len(strings.Split(value, ":")) < 2 {
```

Comment:
let's be strict and use != 2, what do you think?

Reply:
Header value can have a `:` IIRC?

Reply:
makes sense